vendor/init.bat

```diff
@@ -95,9 +126,12 @@
 )
 
 @pushd "%CMDER_ROOT%\config\profile.d"
-for /f "usebackq" %%x in ( `dir /b *.bat *.cmd` ) do (
-  REM @echo Calling %CMDER_ROOT%\config\profile.d\%%x...
-  @call %%x
+@>nul 2>nul dir /b *.bat *.cmd && (
```

Comment:
Is this really necessary, it's confusing?  If there are no files in the profile.d folder then the for loop will not do anything.  You are just running ```dir /b *.bat *.cmd``` twice instead of once in the event there are files.  Why run it twice if once does the job?